src/main/java/com/epam/test_generator/services/utils/UtilsService.java

```diff
@@ -29,6 +25,17 @@
         }
     }
 
+    /**
+     * Checks if Token is null. Returns nothing if not.
+     *
+     * @throws TokenMissingException if Token is null.
+     */
+    public static void checkNotNullToken(Token token) {
```

Comment:
Let's avoid using static methods like that one

Reply:
Is it ok then to check Token for null everywhere we use it? 
Or there is a better solution?

Reply:
Ok. Done.